graphics/opentoonz/README

```diff
@@ -0,0 +1,10 @@
+OpenToonz is a 2D animation software published by DWANGO. It is based
+on Toonz Studio Ghibli Version, originally developed in Italy by
+Digital Video, Inc., and customized by Studio Ghibli over many years
+of production.
+
+After installation, a local "stuff" directory on your user area will
+be required in order for the program to run. This directory can be
+initialised by running the following commands:
+  mkdir -p $HOME/.config/OpenToonz
+  cp -r /opt/opentoonz/share/opentoonz/stuff $HOME/.config/OpenToonz/
```

Comment:
Actually, now that I installed this, the `stuff` directory is not in opt:

```sh
$ cp -r /opt/opentoonz/share/opentoonz/stuff $HOME/.config/OpenToonz/
cp: cannot stat '/opt/opentoonz/share/opentoonz/stuff': No such file or directory
$ cp -r /usr/share/opentoonz/stuff $HOME/.config/OpenToonz/
```

Reply:
`/usr/bin/opentoonz` actually does this step for you (which is what the .desktop file points to) - not sure if it really needs to be in the README.